### ISSUE TITLE
undefined symbol: nc_def_enum

### ISSUE THREAD

**Issue by frank-stonybrook** [Unidata/netcdf4-python#1012]
Dear developers,

I am trying pip(20.1.1) install netCDF5-1.5.3 on a IBM POWER9 system with `gcc/9.1.0,hdf5/1.10.3` and `netcdf/4.6.1`. However, running `python -c "import netCDF4"` causes the following error:
`ImportError: /autofs/nccs-svm1_proj/bip204/src_cong/Virtualenv/CASP14/lib/python3.7/site-packages/netCDF4/_netCDF4.cpython-37m-powerpc64le-linux-gnu.so: undefined symbol: nc_def_enum`

Thanks for your help!

**Comment by jswhit** [comment]
try `ldd` on the _netCDF4.cpython-37m-powerpc64le-linux-gnu.so file and see if the netcdf library shows up

**Comment by frank-stonybrook** [comment]
Yes, I did try that and everything seems normal. Later I fixed it by manually building and installing it via the `setup.py` script. Comparing the two _netCDF4.cpython-37m-powerpc64le-linux-gnu.so files, the one I got from the second method has much more entries. I guess the `pip` installed under my virtual environment seems failed to link lots of system-wide libraries. Anyhow, it got fixed. Thanks
